src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/CheckBoxTests.cs

```diff
@@ -527,4 +530,126 @@ internal override bool RaiseAutomationPropertyChangedEvent(UIA_PROPERTY_ID prope
             return base.RaiseAutomationPropertyChangedEvent(propertyId, oldValue, newValue);
         }
     }
+
+    [WinFormsFact]
+    public void CheckBox_CheckedChangedEvent_Fires()
```

Comment:
```suggestion
    public void CheckBox_CheckedChangedEvent_Raised()
```

And the same applies to the variable name and the next test name